server/sublist.go

```diff
@@ -92,7 +92,8 @@ func newLevel() *level {
 
 // New will create a default sublist
 func NewSublist() *Sublist {
-	return &Sublist{root: newLevel(), cache: make(map[string]*SublistResult)}
+	//	return &Sublist{root: newLevel(), cache: make(map[string]*SublistResult)}
```

Comment:
Is that intentional? If you don't create the cache here, it will never be created therefore used.

Reply:
This was me playing with other things, should not be in here. Will remove.